### PR TITLE
Update the copy of the right request emails

### PR DESCRIPTION
**Pull request by bmesuere**
This pull request updates the copy of the right request emails.

The text is now a bit more customer focussed since this is one of the first contacts we have with teachers. In addition, I added links to the demo video and to our twitter and facebook accounts (in Dutch only).

@chvp if you're not ok with your name being in the emails, just let me know and I'll replace it with mine.

### CODE REVIEW

File: app/views/rights_request_mailer/approved.en.html.erb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <div class="message">
-    <p>Hi <%= @request.user.full_name %>,</p>
+    <p>Hi <%= @request.user.first_name %>,</p>
```
Comment (chvp) [inline app/views/rights_request_mailer/approved.en.html.erb:2]: `first_name` can sometimes be empty, so I'd add an `||` here with the full name.

File: app/views/rights_request_mailer/approved.nl.text.erb [hunk 2]
```diff
@@ -1,12 +1,13 @@
-Dag <%= @request.user.full_name %>
+Dag <%= @request.user.first_name %>
 
-Goed nieuws! Je Dodona-account heeft lesgeversrechten gekregen. Je kan
-nu cursussen aanmaken en oefeningen aan het platform toevoegen. Een
-handleiding over hoe van start te gaan kan je hier vinden:
-https://docs.dodona.be/nl/guides/teachers/getting-started/.
+Goed nieuws! We hebben je Dodona-account lesgeversrechten gegeven. Je kan nu cursussen aanmaken en zelf oefeningen aan Dodona toevoegen.
```
Comment (chvp) [inline app/views/rights_request_mailer/approved.nl.text.erb:3]: The few people who see the plaintext version will probably appreciate it if we wrap this properly.

**Review by chvp — CHANGES_REQUESTED** [review]
No personalized name for the reject?

File: app/views/rights_request_mailer/approved.nl.html.erb [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
+    <p>
+        Toch nog vragen? Stuur gerust een mailtje naar <a href="mailto:<%= Rails.application.config.dodona_email %>"><%= Rails.application.config.dodona_email %></a>.
     </p>
     <p>
         Met vriendelijke groeten,<br/>
+        Charlotte<br/>
         Team Dodona
     </p>
+    <p>
+        ps. Volg ons snel op <a href="https://twitter.com/DodonaEdu">Twitter</a> of <a href="https://facebook.com/DodonaEdu">Facebook</a> en wees als eerste op de hoogte van onze nieuwste functies.
```
Comment (niknetniko) [inline app/views/rights_request_mailer/approved.nl.html.erb:18]:
```suggestion
        PS Volg ons snel op <a href="https://twitter.com/DodonaEdu">Twitter</a> of <a href="https://facebook.com/DodonaEdu">Facebook</a> en wees als eerste op de hoogte van onze nieuwste functies.
```

File: app/views/rights_request_mailer/approved.nl.text.erb [hunk 4]
```diff
@@ -1,12 +1,19 @@
...
+Peter je op https://www.youtube.com/watch?v=IfXrsP1z9OI&t=698s een uitgebreid
+overzicht van hoe je Dodona in jouw klas kan gebruiken.
+
+Toch nog vragen? Stuur gerust een mailtje naar <%= Rails.application.config.dodona_email %>.
 
 Met vriendelijke groeten,
+Charlotte
 Team Dodona
+
+ps. Volg ons snel op Twitter (https://twitter.com/DodonaEdu) of Facebook
```
Comment (niknetniko) [inline app/views/rights_request_mailer/approved.nl.text.erb:17]:
```suggestion
PS Volg ons snel op Twitter (https://twitter.com/DodonaEdu) of Facebook
```

**Review by niknetniko — CHANGES_REQUESTED** [review]

**Review by niknetniko — APPROVED** [review]

**Pull request merged**